ee/clickhouse/queries/funnels/base.py

```diff
@@ -397,30 +397,24 @@ def _get_cohort_breakdown_join(self) -> str:
 
     def _get_breakdown_conditions(self) -> str:
         if self._filter.breakdown:
-            if self._filter.funnel_step_breakdown:
-                values = self._parse_breakdown_prop_value()
-            else:
-                limit = self._filter.breakdown_limit_or_default
-                first_entity = next(x for x in self._filter.entities if x.order == 0)
-                if not first_entity:
-                    ValidationError("An entity with order 0 was not provided")
-                values = []
-                if self._filter.breakdown_type == "person":
-                    values = get_breakdown_person_prop_values(
-                        self._filter, first_entity, "count(*)", self._team.pk, limit
-                    )
-                elif self._filter.breakdown_type == "event":
-                    values = get_breakdown_event_prop_values(
-                        self._filter, first_entity, "count(*)", self._team.pk, limit
-                    )
+            limit = self._filter.breakdown_limit_or_default
+            first_entity = next(x for x in self._filter.entities if x.order == 0)
+            if not first_entity:
+                ValidationError("An entity with order 0 was not provided")
+            values = []
+            if self._filter.breakdown_type == "person":
+                values = get_breakdown_person_prop_values(self._filter, first_entity, "count(*)", self._team.pk, limit)
+            elif self._filter.breakdown_type == "event":
+                values = get_breakdown_event_prop_values(self._filter, first_entity, "count(*)", self._team.pk, limit)
```

Comment:
So what happens if the second entity has a different set of person property values? Is it impossible due to how funnels work?

If so, worth adding a comment.

Reply:
What do you mean by different set of person property values for second entity? Maybe an example?

If you mean something like: breakdown by X, and second entity has a filter like: X=a, where a is very rare, then earlier, yes, this would mean that drop off is 100%. After this PR, they'd all get grouped into "Other".

Reply:
So entities in this case are things like:
1. { type: event, value: 'fooevent' }
2. { type: event, value: 'barevent' }

Let's walk through a scenario 
If you break down first with `$some_property` you get values A, B. If you broke the second with `$some_property` you would get B and C

Would the table contain A and B only or would C get included as well or is this scenario impossible?

Reply:
I hope you see now why this needs at minimum a clarifying comment - there's a ton to unpack here.

Why `Other` not C? 3 values will fall well below the line of 5 or 10. :)

Given you can break down by event properties, it's more than possible that the second step in the funnel has a different set of properties than the first. It's kind of how event properties work - they don't stick around for all events for a session.

Reply:
Ah, I see what you mean now, "Why other not C" is a very valid question!

Reply:
oh wait, actually, in the funnel, all event properties are constrained to the first in line, so even with other grouping, the breakdown values would only be `A, B`, no other. (will write a test to confirm). My bad here, I got confused: the query never does anything with props for events following the first one. 

I'll chalk this one up to a funnel limitation. With event props, makes sense to extend this like you say.

@EDsCODE can confirm?